CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ option(WITH_TESTSCRIPTS       "Set to ON to run CMakeLists.txt in testscripts"
 option(WITH_GPROF             "Set -pg to compile flags"                                                         OFF)
 option(WITH_DLTTEST           "Set to ON to build with modifications to test User-Daemon communication with corrupt messages" OFF)
 option(WITH_DLT_SHM_ENABLE    "EXPERIMENTAL! Set to ON to use shared memory as IPC. EXPERIMENTAL!"               OFF)
-option(WITH_DLT_ADAPTOR       "Set to ON to build src/adaptor binaries"                                          OFF)
```

Comment:
Can you keep this cmake option so that we can keep backward compatibility?

Reply:
yes, sure. I re-added the option. Now you can build all with `WITH_DLT_ADAPTOR` or disable single items